index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+    <title>My portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&family=Roboto:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&family=Roboto:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+        <nav>
+        <a href="">Divine A.</a>
+         <img src="Images/532-5328945_menu-bar-icon-white-clipart-png-download-menu.png" alt=""> 
+        </nav>
+        <header >
+                <h1>Hey There.
+                    I’m Divne
+                </h1>
+            <h2>
+                I am a Software Developer
+            </h2>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+            </p>
+            <div>
+               <img src="Images/Social Media.png" alt="Social icons">
+            </div>
```

Comment:
- [ ] As the previous reviewer mentioned Kindly wrap this part into a `section` instead of a `header` to make it semantically correct. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.